[2024-02-20T13:06:00.494+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: airflow-s3-mlflow.download_from_s3 manual__2024-02-20T13:05:53.352639+00:00 [queued]>
[2024-02-20T13:06:00.529+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: airflow-s3-mlflow.download_from_s3 manual__2024-02-20T13:05:53.352639+00:00 [queued]>
[2024-02-20T13:06:00.533+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 1
[2024-02-20T13:06:00.561+0000] {taskinstance.py:2191} INFO - Executing <Task(PythonOperator): download_from_s3> on 2024-02-20 13:05:53.352639+00:00
[2024-02-20T13:06:00.581+0000] {standard_task_runner.py:60} INFO - Started process 949 to run task
[2024-02-20T13:06:00.589+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', '***-s3-mlflow', 'download_from_s3', 'manual__2024-02-20T13:05:53.352639+00:00', '--job-id', '12', '--raw', '--subdir', 'DAGS_FOLDER/***-aws-mlflow.py', '--cfg-path', '/tmp/tmpoyqioq8w']
[2024-02-20T13:06:00.593+0000] {standard_task_runner.py:88} INFO - Job 12: Subtask download_from_s3
[2024-02-20T13:06:00.619+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.8/site-packages/***/settings.py:194 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2024-02-20T13:06:00.699+0000] {task_command.py:423} INFO - Running <TaskInstance: airflow-s3-mlflow.download_from_s3 manual__2024-02-20T13:05:53.352639+00:00 [running]> on host eb0fadca60cf
[2024-02-20T13:06:00.859+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='***-s3-mlflow' AIRFLOW_CTX_TASK_ID='download_from_s3' AIRFLOW_CTX_EXECUTION_DATE='2024-02-20T13:05:53.352639+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-02-20T13:05:53.352639+00:00'
[2024-02-20T13:06:00.863+0000] {s3.py:1375} INFO - This function shadows the 'download_file' method of S3 API, but it is not the same. If you want to use the original method from S3 API, please call 'S3Hook.get_conn().download_file()'
[2024-02-20T13:06:00.864+0000] {s3.py:1381} INFO - Downloading source S3 file from Bucket airdemobkt with path wine-quality.csv
[2024-02-20T13:06:00.879+0000] {base_aws.py:581} WARNING - Unable to find AWS Connection ID 's3_conn', switching to empty.
[2024-02-20T13:06:00.881+0000] {base_aws.py:161} INFO - No connection ID provided. Fallback on boto3 credential strategy (region_name=None). See: https://boto3.amazonaws.com/v1/documentation/api/latest/guide/configuration.html
[2024-02-20T13:06:01.323+0000] {credentials.py:1147} INFO - Found credentials in environment variables.
[2024-02-20T13:06:08.591+0000] {python.py:201} INFO - Done. Returned value was: /opt/***/data/***_tmp_be24307w
[2024-02-20T13:06:08.621+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=***-s3-mlflow, task_id=download_from_s3, execution_date=20240220T130553, start_date=20240220T130600, end_date=20240220T130608
[2024-02-20T13:06:08.662+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-02-20T13:06:08.694+0000] {taskinstance.py:3280} INFO - 1 downstream tasks scheduled from follow-on schedule check
